pkgs/top-level/all-packages.nix

```diff
@@ -16900,6 +16900,8 @@ with pkgs;
     llvmPackages = llvmPackages_latest;
   };
 
+  coq-lsp = callPackage ../development/tools/language-servers/coq-lsp { };
```

Comment:
Should be in `coq-packages.nix` instead due to the dependency on a version of Coq.